payloads/dynamic-base-scratch/runenv.dockerfile

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright 2021 Kontain Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM scratch
+ENV KM_DO_SHELL NO
+ADD --chown=0:0 libs.tar /opt/kontain/
+ADD empty_tmp /tmp/
```

Comment:
wonder if there's a way to avoid this step.  Looks weird in a Dockerfile.

Reply:
This is our base Dockerfile, not really visible to users. We do need to make sure `/tmp/` is there. Many things don't work without it.

In theory we could make the tar image with it and simply ADD that. Let me know how strongly you feel about it.